src/units/store.rs

```diff
@@ -169,3 +170,140 @@ impl<'a, H: Hasher, D: Data, KB: KeyBox> UnitStore<'a, H, D, KB> {
         self.max_round
     }
 }
+
+#[cfg(test)]
```

Comment:
I think we prefer having tests in the `testing` directory. At least I do, makes compiling quicker...

Reply:
We have such tests in other files as well. I thought it makes more sense for unit tests of UnitStore struct to be in the UnitStore file while in `testing` directory tests usually are less unit like and more integration like. Regarding compilation times the `#[cfg(test)]` annotation prevents compilation unless something like `cargo test` is run - so commands like `cargo build` should not be affected - https://doc.rust-lang.org/book/ch11-03-test-organization.html#the-tests-module-and-cfgtest.

Reply:
If you change something in a file, then the file and anything that depends on it has to be recompiled, even if you only changed the test part. :<

And I know about the other tests in files, that's why I'm complaining, it's from experience. D:

Technically this is not a blocker, although I would like to unify the way we do tests, and preferably in the separate-file manner for reasons outlined above. The tests in the `testing` directory are definitely not only integration-like, at least the alert tests are the most unit-testy I could come up with.

Reply:
I agree with @timorl that we should standardize the place for tests. Currently, the effort was to put unit tests in corresponding files and intergration tests in testing, so this PR fits into that effort. We could move all the tests to testing but let's do it in a separate PR.